apps/prairielearn/src/pages/instructorAssessmentInstances/instructorAssessmentInstances.sql

```diff
@@ -117,13 +118,7 @@ WITH
               WHEN $base_time = 'current_date' THEN current_timestamp
               ELSE ai.date_limit
             END
-          ) + (
-            CASE
-              WHEN $time_ref = 'minutes' THEN make_interval(mins => $time_add)
-              WHEN $time_ref = 'percent' THEN (ai.date_limit - ai.date) * $time_add / 100
```

Comment:
Say more about why you removed percent-based updates? @mwest1066 do you think there's value in keeping this functionality around?

Reply:
I thought we were removing this option in place of making 'minutes' a fixed suffix rather than an option from this issue (#10727). This code was then unused so I removed it. I can keep the percent-based updates if you would prefer.

Reply:
Apologies, it was only in the specific case I was testing (operating on a single, closed assessment instance) where the dropdown only had a "minutes" option. If the assessment instance had time remaining, or if operating on all assessment instances where at least one has time remaining, the "percent" option was still shown.

Regardless of all this, I'm still checking with Matt to see if we want to retain the percentage-based adjustments in those cases.